src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -401,25 +420,18 @@ internal sealed partial class AssignmentExpression : BaseAssignmentExpression, I
         /// Value to be assigned to the target of the assignment.
         /// </summary>
         public override IOperation Value { get; }
-        public override void Accept(OperationVisitor visitor)
-        {
-            visitor.VisitAssignmentExpression(this);
-        }
-        public override TResult Accept<TArgument, TResult>(OperationVisitor<TArgument, TResult> visitor, TArgument argument)
-        {
-            return visitor.VisitAssignmentExpression(this, argument);
-        }
     }
 
     /// <summary>
     /// Represents an assignment expression.
     /// </summary>
-    internal sealed partial class LazyAssignmentExpression : BaseAssignmentExpression, IAssignmentExpression
+    internal sealed partial class LazySimpleAssignmentExpression : BaseSimpleAssignmentExpression, ISimpleAssignmentExpression
     {
         private readonly Lazy<IOperation> _lazyTarget;
```

Comment:
we don't want to realize all child at some for example blocks. this Lazy pattern is ver 1. so that we can share one implementation for both C# and VB. we might later move this to C#/VB explicitly type that directly hold on to bound node that implements common IOperation interface for laziness.

we decide this way since interface is the public API and implementation we can change later.